src/content/documentation/publish/signing-and-distribution-overview.md

```diff
@@ -75,8 +76,10 @@ Mozilla signs add-ons through [addons.mozilla.org](https://addons.mozilla.org).
 
 | Signing method     | Supported distribution channel(s) | 
 | ------------------------------------- | ------------------- | 
-| Web upload via the [AMO Developer Hub](https://addons.mozilla.org/developers/) | Public listing on AMO or self-distribution| 
-| Submit using [web-ext sign](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign) or using the [AMO signing API](https://addons-server.readthedocs.io/en/latest/topics/api/v4_frozen/signing.html) | Brand new submissions can only be submitted as self-distributed (unlisted) extensions. <br /><br /> Subsequent updates can be listed on AMO or self-distributed (unlisted)| 
+| Web upload via the [AMO Developer Hub](https://addons.mozilla.org/developers/) | Public listing on AMO or self-distribution (unlisted). | 
+| Upload using [web-ext sign](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign) | <strong>Version 8 and higher:</strong> Public listing on AMO or self-distribution (unlisted). <br /><br /><strong>Version 7 and lower:</strong> New add-on submissions can only be submitted as self-distributed (unlisted) extensions. Subsequent add-on updates can be listed on AMO or self-distributed (unlisted). | 
```

Comment:
Unless necessary stick to markdown?
```suggestion
| Upload using [`web-ext sign`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign) | **Version 8 and higher**: Public listing on AMO or self-distribution (unlisted). <br /><br />**Version 7 and lower**: Initial submissions only provide for self-distributed (unlisted) extensions. Updates can be for extensions listed on AMO or self-distributed (unlisted). | 
```

Reply:
I'm not sure why I used HTML instead of Markdown here. I'll go with your suggestion.